mods/species/bayliens/adherent/organs/organs_internal.dm

```diff
@@ -10,13 +10,17 @@
 	var/next_rename
 	var/rename_delay = 15 MINUTES
 
-/obj/item/organ/internal/brain/adherent/refresh_action_button()
+/obj/item/organ/internal/posibrain/adherent/Initialize()
+	BP_SET_CRYSTAL(src)
+	. = ..()
+
+/obj/item/organ/internal/posibrain/adherent/refresh_action_button()
 	. = ..()
 	if(.)
 		action.button_icon_state = "adherent-brain"
 		if(action.button) action.button.UpdateIcon()
 
-/obj/item/organ/internal/brain/adherent/attack_self(var/mob/user)
+/obj/item/organ/internal/posibrain/adherent/attack_self(var/mob/user)
```

Comment:
Adherent are not positronic, this is not a good repath imo.

Reply:
Well, its hard to tell from the weird conflicting info around the code. Some of the stuff says they're piezo-electronic. I can set them back to an organic brain though. With the changes in here it won't instantly kill them anymore at least.